### DISCUSSION TITLE
Compiling error in Win11 using option -w

### DISCUSSION THREAD

**Discussion by JohnVanMersbergen** [thonny/thonny#3198] [General]
Since a few week I got a compiling error in Windows 11 (23H2) using PYinstaller with option -w. According to Windows Defender it concerns the virus Trojan:Win64/Malgent!MSR. 

![Python_virus](https://github.com/thonny/thonny/assets/144234863/11a912ed-449d-4afe-83b2-1da3783906f9)

![WD_melding](https://github.com/thonny/thonny/assets/144234863/acab4f01-54e8-47e5-b8a8-51d1583b193b)

**Comment by aivarannamaa** [comment]
I suppose it's better to take this issue to a Pyinstaller forum. Most likely it is a false alarm, though. Here is a discussion, which implies that false virus alarms are common with Pyinstaller: https://stackoverflow.com/questions/43777106/program-made-with-pyinstaller-now-seen-as-a-trojan-horse-by-avg
